summerofcode/2023.md

```diff
@@ -26,12 +26,26 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 
 ## Ideas
 
+- [DevStream](#devstream)
+  - [Developer Tool Improvement](#developer-tool-improvement)
 - [Knative](#knative)
   - [Multiple Knative Eventing control planes](#multiple-knative-eventing-control-planes)
   - [Eventing Sender Identity](#eventing-sender-identity)
   - [NetworkPolicy support for Knative Serving](#networkpolicy-support-for-knative-serving)
   - [Improving Observability in Knative Eventing](#improving-observability-in-knative-eventing)
 
+### DevStream
+
+#### Developer Tool Improvement
+
+- Description: DevStream wants to build a developer tool that can bootstrap your local dev env at the click of a button using interactive CLI. Currently we are doing MVP and market researches. We want to add more support regarding more programming languages and frameworks and CI/CD tools so that the users can bootstrap a new microservice using predefined template and setup the whole CI/CD pipelines locally.
```

Comment:
If this project is not about doing the MVP and the market research itself, IMO it is better to specify which languages/frameworks/tools you would like a contributor to implement the support for.

I wanted to mention this because I think it is important for a potential contributor to assess their knowledge for the requirements of this project. For example, they might know a language/framework/tool you wouldn't need in your community.